pandas/tests/frame/conftest.py

```diff
@@ -5,6 +5,23 @@
 import pandas.util.testing as tm
 
 
+@pytest.fixture
+def empty_frame():
+    """
+    Fixture for DataFrame that is empty
+    """
+    return DataFrame()
+
+
+@pytest.fixture
+def float_frame():
```

Comment:
there *already* is a float_frame fixture (defined in pandas/conftest.py), don't duplicate

Reply:
Perfect - I couldn't find it before